app/controllers/users.rb

```diff
@@ -0,0 +1,77 @@
+get '/users' do
+  @users = User.all
+  erb :'users/all_users'
+end
+
+# Page to create a new user
+get '/users/new' do
+  @errors = {}
+  erb :'users/new_user'
+end
+
+# Authenticate and update the session / Return errors
+post '/users/login' do
+  # Handle when the user does not exist
+  @incorrect_login = false
+  login
```

Comment:
You should at least pass params into your helper. Something like `login(params[:user_name], params[:password])`